CHANGELOG.md

```diff
@@ -24,6 +24,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Changed
 
 - Update Jest major version
+- Changed `MeetingManager` to strictly enforce `DevicePermissionStatus` type.
```

Comment:
It would be helpful to have changelog for the export of DevicePermissionStatus hook.